### PR TITLE
finished second level "B" spells

### PR DESCRIPTION
**Pull request by ChrisSpangler**
finished the second level "b" spells

### CODE REVIEW

File: _posts/spells/cantrips/2020-10-13-spells-2-augury.md
Comment (magicalmusings) [inline _posts/spells/cantrips/2020-10-13-spells-2-augury.md]: Divination (ritual)

File: _posts/spells/cantrips/2020-10-13-spells-2-augury.md
Comment (magicalmusings) [inline _posts/spells/cantrips/2020-10-13-spells-2-augury.md]: Players Handbook or PHB

File: _posts/spells/cantrips/2020-10-14-spells-2-beast-sense.md
Comment (magicalmusings) [inline _posts/spells/cantrips/2020-10-14-spells-2-beast-sense.md]: Divination (ritual)

**Review by magicalmusings — CHANGES_REQUESTED** [review]
These are great! just a couple minor edits I mentioned in comments, and a couple of points here:

- All of these spells are actually from the Players Handbook ,or PHB. On the https://5e.tools/spells.html#branding%20smite_phb you can see the source on there in either the table or the spell card itself. 
- All of these spells are currently in the _posts/spells/cantrips/ folder, just move them to _post/spells/second-level/ folder 

Other than that, ready to approve and merge when those changes are made! Thanks @ChrisSpangler , let me know if you got any questions or concerns.

**Comment by ChrisSpangler** [comment]
Added the corrections - wasn't sure if you wanted "Players Handbook" or "PHB" or "Players Handbook or PHB" (together)

Let me know and I will make the fixes!

**Comment by magicalmusings** [comment]
Probably just 'Player's Handbook (PHB)' or 'Player's Handbook' is fine @ChrisSpangler

**Review by magicalmusings — CHANGES_REQUESTED** [review]
Like I said via comment, just change to either 'Player's Handbook (PHB)' or 'Player's Handbook' and we'll be good to merge!

**Comment by ChrisSpangler** [comment]
Changes made :) - thank you for your patience

**Review by magicalmusings — APPROVED** [review]

**Comment by magicalmusings** [comment]
No worries! I am pretty new to Github as well, I only know how to handle these PRs because of my Open Source Software class in college. Apologies on any weird notifications, was setting up some Continuous Integration to automatically check whether things are not going to break the site before I merge pull requests / commits. 

Thanks for your help @ChrisSpangler!

**Pull request merged**
